frontend/public/index.html

```diff
@@ -2,13 +2,10 @@
 <html lang="en">
```

Comment:
Can we also generate the correct files that are usually used for favicons on the web?

See the linked article in the original issue: https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs

Reply:
May I clarify what you mean by correct files? Am I required to follow all the steps in the linked article or do I just have to generate the following files:
`logo192.png`
`logo512.png`
`apple-touch-icon.png`
`favicon.ico`
`favicon.svg`
Do let me know if I missed out anything thanks!

Reply:
Yeah, just generate the files; ensure they are the correct size or minified etc. The article is just a guide